kuma/wiki/search.py

```diff
@@ -288,86 +287,31 @@ def reindex_all(cls, chunk_size=1000, index=None, percent=100):
 
         """
         from kuma.search.models import Index
+        from kuma.search.tasks import prepare_index, finalize_index
         from kuma.wiki.tasks import index_documents
 
         index = index or Index.objects.get_current()
-        index_name = index.prefixed_name
-
-        es = cls.get_connection('indexing')
-
-        log.info('Wiping and recreating %s....', index_name)
-        Index.objects.recreate_index(es=es, index=index)
 
+        # Get the list of document IDs to index.
         indexable = WikiDocumentType.get_indexable(percent)
 
-        # We're doing a lot of indexing, so we get the refresh_interval
-        # currently in the index, then nix refreshing. Later we'll restore it.
-        index_settings = {}
-        try:
-            index_settings = (es.indices.get_settings(index_name)
-                                .get(index_name, {}).get('settings', {}))
-        except NotFoundError:
-            pass
-
-        refresh_interval = index_settings.get(
-            'index.refresh_interval', settings.ES_DEFAULT_REFRESH_INTERVAL)
-        number_of_replicas = index_settings.get(
-            'number_of_replicas', settings.ES_DEFAULT_NUM_REPLICAS)
-
-        # Disable automatic refreshing.
-        temporary_settings = {
-            'index': {
-                'refresh_interval': '-1',
-                'number_of_replicas': '0',
-            }
-        }
+        total = len(indexable)
+        total_chunks = int(ceil(total / float(chunk_size)))
```

Comment:
This file already has future division in it. So I'll just remove the `float` wrapper there.